specs/simple-serialize.md

```diff
@@ -37,71 +37,96 @@
 ## Typing
 ### Basic types
 
-* `"uintN"`: `N`-bit unsigned integer (where `N in [8, 16, 32, 64, 128, 256]`)
-* `"bool"`: `True` or `False`
+* `uintN`: `N`-bit unsigned integer (where `N in [8, 16, 32, 64, 128, 256]`)
+* `boolean`: `True` or `False`
 
 ### Composite types
 
 * **container**: ordered heterogeneous collection of values
-    * key-pair curly bracket notation `{}`, e.g. `{"foo": "uint64", "bar": "bool"}`
-* **vector**: ordered fixed-length homogeneous collection of values
-    * angle bracket notation `[type, N]`, e.g. `["uint64", N]`
-* **list**: ordered variable-length homogeneous collection of values
-    * angle bracket notation `[type]`, e.g. `["uint64"]`
+    * python dataclass notation with key-type pairs, e.g.
+```python
+class ContainerExample(Container):
+    foo: uint64
+    bar: boolean
+```
+* **vector**: ordered fixed-length homogeneous collection, with `N` values
+    * notation `Vector[type, N]`, e.g. `Vector[uint64, N]`
+* **list**: ordered variable-length homogeneous collection, limited to `N` values
+    * notation `List[type, N]`, e.g. `List[uint64, N]`
+* **bitvector**: ordered fixed-length collection of `boolean` values, with `N` bits
+    * notation `Bitvector[N]`
```

Comment:
`BitVector` and `BitList` (capitalize first letter of second word) are more pythonic names for types. 
What do you think about changing this?